### DISCUSSION TITLE
Sonoff POW R3 25A 5500W Wont Boot when Sensor Connected

### DISCUSSION THREAD

**Discussion by juan11perez** [arendst/Tasmota#17732] [Q&A] [answered]
Flashed Sonoff POW R3 25A 5500W with tasmota and connected a dht220 to GPIO15, 3.3 and Ground.

While sensor 3.3 is connected to Sonoff POW R3 25A 5500W board it wont start. 

If I disconnect the sensor 3.3 from Sonoff POW R3 25A 5500W the unit will start. After start up I can connect the sensor 3.3v and it works as expected.

**Comment by sfromis** [comment, marked as answer]
Answer removed, since it is dangerous to connect anything to an power monitoring device.
NEVER do this!

**Reply by juan11perez** [reply]
thank you. It has available 15 and 16. Should i use 16?
or rx?

**Reply by sfromis** [reply]
You should not see such devices as recommended for adding stuff to. And rx is not free anyway. But if you want to try some other pin, it is your device :wink:

**Reply by juan11perez** [reply]
Correct rx not free. Thank you for the advise.

**Reply by barbudor** [reply]
And your life.. 
POW devices are know to have CPU pins at AC level
Beware where you put your fingers

We generally discourage tinkering with power measurement devices
